simvue/run.py

```diff
@@ -45,72 +48,107 @@
     validate_timestamp,
 )
 
-INIT_MISSING = "initialize a run using init() first"
-QUEUE_SIZE = 10000
-UPLOAD_TIMEOUT = 30
+if typing.TYPE_CHECKING:
+    from .factory.base import SimvueBaseClass
+
+UPLOAD_TIMEOUT: int = 30
 HEARTBEAT_INTERVAL: int = 60
 RESOURCES_METRIC_PREFIX: str = "resources"
 
 logger = logging.getLogger(__name__)
 
 
-def walk_through_files(path):
-    for dirpath, _, filenames in os.walk(path):
+def walk_through_files(
```

Comment:
This is a tiny, trivial function that is only used in one place. I think it should be inlined.